### PR TITLE
6927: updated button for description

### PR DESCRIPTION
**Pull request by nikola-gladovic**
Signed-off-by: Nikola <nikola.gladovic@nextcloud.com>

#6927

### CODE REVIEW

File: src/components/Description/Description.vue [hunk 1]
```diff
@@ -35,21 +35,27 @@
...
 					@click="handleCancelEditing">
-					<Close decorative
-						title=""
-						:size="16" />
-				</button>
-				<button class="nc-button nc-button__main primary description__action"
+					<template #icon>
+						<Close decorative
+							title=""
+							:size="16" />
```
Comment (nickvergessen) [inline src/components/Description/Description.vue:45]: Button seems to not be visible... not sure what's wrong

Comment (nickvergessen) [reply src/components/Description/Description.vue:45]: Bildschirmfoto vom 2022-03-14 12-01-59

Comment (marcoambrosini) [inline src/components/Description/Description.vue:45]: Also the size of the icons should always be 20, could you please change that?

File: src/components/Description/Description.vue [hunk 2]
```diff
@@ -35,21 +35,27 @@
...
 					:disabled="!canSubmit"
 					@click="handleSubmitDescription">
-					<Check decorative
-						title=""
-						:size="16" />
-				</button>
+					<template #icon>
+						<Check decorative
+							title=""
+							:size="16" />
```
Comment (marcoambrosini) [inline src/components/Description/Description.vue:56]: Same here

File: src/components/Description/Description.vue [hunk 3]
```diff
@@ -58,14 +64,14 @@
...
 			</template>
-			<button v-if="!editing && editable"
-				class="nc-button nc-button__main"
+			<Button v-if="!editing && editable"
+				type="tertiary"
 				:aria-label="t('spreed', 'Edit conversation description')"
 				@click="handleEditDescription">
 				<Pencil decorative
 					title=""
 					:size="16" />
```
Comment (marcoambrosini) [inline src/components/Description/Description.vue:73]: And here

File: src/components/Description/Description.vue (hunk 1 above)
Comment (marcoambrosini) [reply src/components/Description/Description.vue:45]: for some reason hasIcon is false and text true 🤔 

Screenshot 2022-03-15 at 09 57 59

Comment (nikola-gladovic) [reply src/components/Description/Description.vue:45]: Ok

Comment (nikola-gladovic) [reply src/components/Description/Description.vue:45]: Umm, let me check it.

Comment (nickvergessen) [reply src/components/Description/Description.vue:45]: Maybe from the broken lib?

Comment (nikola-gladovic) [reply src/components/Description/Description.vue:45]: Yes, I think so.

**Comment by marcoambrosini** [comment]
A `template #icon` was missing

https://user-images.githubusercontent.com/26852655/158609057-df12f9c0-4884-4de5-8007-84d3771c9ce9.mov

**Review by marcoambrosini — APPROVED** [review]

**Pull request merged**